tut/stream-client.md

```diff
@@ -0,0 +1,174 @@
+This tutorial guides you through the process of setting up a web-based streaming client using the Anbox Cloud streaming stack.
+
+## Preparation
```

Comment:
Setup? Prerequisites?
I'm not against Preparation, I just wonder if something else would sound better.

Reply:
Makes sense!